syntax = "proto3";

message Numbers {
    int64 x = 1;
    int64 y = 2;
}

message Result {
    int64 z = 3;
}

message ImgInput {
    bytes img = 1;
}

message ImgOutput {
    float width = 1;
    float height = 2;
}

service Service1 {
    rpc AddNumbers(Numbers) returns (Result) {}
    rpc ProcessImg(ImgInput) returns (ImgOutput) {}
}